### DISCUSSION TITLE
Google Docs gview shows "No Cookie" for PHP generated via PHP using mPDF

### DISCUSSION THREAD

**Discussion by anjanesh** [mpdf/mpdf#1713] [Q&A]
my-document.php generates a PDF using PHP via mPDF
```
<?php
$pdf = CB_ABS_URI."my-document.php?ID=".$_GET['ID'];
$gview = "https://docs.google.com/gview?url=".urlencode($pdf)."&embedded=true";
echo $gview;
?>
<iframe src="<?php echo $gview ?>" style="width:950px;height:750px;" frameborder="0"></iframe>
```

Im getting a preview which shows "No Cookie"

<img width="203" alt="no-cookie" src="https://user-images.githubusercontent.com/85973/188255001-875cc97e-b567-45d2-829d-d80573563025.png">

What should I be doing in my-document.php to get this recognized as a valid PDF for google's embedded view ?
Set cookie headers ?

**Comment by finwe** [comment]
I have no idea what you are trying to do nor what your question is. I can't even reproduce your code.

**Comment by anjanesh** [comment]
My apologies - my PDF generator using mPDF uses cookies to check for logged in users only access. Hence the script at my-document.php was returning a No Cookies error because it wasn't getting sent through to google's doc viewer. My bad.

**Reply by finwe** [reply]
Does that mean you solved it?
